content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/image.html

```diff
@@ -26,12 +26,24 @@
        class="cmp-image__link"
        data-sly-attribute="${image.imageLink.htmlAttributes}"
        data-cmp-clickable="${image.data ? true : false}">
-        <img srcset="${image.srcset}" src="${image.src}"
+        <img data-sly-test="${!image.SrcsetWithMimeType}" srcset="${image.srcset}" src="${image.src}"
              loading="${image.lazyEnabled ? 'lazy' : ''}"
              class="cmp-image__image"
              itemprop="contentUrl"
              width="${image.width}" height="${image.height}"
              alt="${image.alt || true}" title="${image.displayPopupTitle && image.title}"/>
+
+        <picture data-sly-test="${image.SrcsetWithMimeType}">
```

Comment:
To avoid duplicating the `img` tag, we could `data-sly-unwrap` the parent `picture` when we don't need to have multiple sources

Reply:
@vladbailescu if img tag is inside the picture tag we do not need srcset attribute and but without picture tag we do need the srcSet Attribute as well in img tag and hence the duplication.